hack/build/go_container.sh

```diff
@@ -73,9 +73,9 @@ run_in_go_container() {
       -e GOOS="${GOOS}" \
       -e GOARCH="${GOARCH}" \
     `# pass through proxy settings from the host` \
-      -e HTTP_PROXY="${HTTP_PROXY:+}" \
-      -e HTTPS_PROXY="${HTTPS_PROX:+}" \
```

Comment:
Nice catch on this typo!
We still need a proxied test-env to try these things out in...